Dockerfile.tmpl

```diff
@@ -24,8 +34,10 @@ RUN sed -i "s/httpredir.debian.org/debian.uchicago.edu/" /etc/apt/sources.list &
     apt-get install -y openssh-client && \
     /tmp/clean-layer.sh
 
+# TODO(rosbo): Is this needed? Make sure gpu and stubs are properly set.
```

Comment:
Are you planning to close this TODO in this PR or test after?

Reply:
Good catch, I removed the TODO since I confirmed this is not needed.